modules/@apostrophecms/ai-helper-rich-text-widget/index.js

```diff
@@ -0,0 +1,88 @@
+const marked = require('marked');
+
+// Canned sample results for cheap & offline testing
+const mockResults = {
```

Comment:
Minor comment: could this go into a external file?

Reply:
At some point